.github/main.workflow

```diff
@@ -0,0 +1,42 @@
+workflow "Build and deploy the documentation" {
+  on = "push"
+  resolves = ["doc-build-push"]
+}
+
+action "doc-check-user" {
+  uses = "./scripts/actions/if-user"
+  args = "^(?!yarnbot$)"
+}
+
+action "doc-check-branch" {
```

Comment:
Why not using the `filter` action provided by Github?
https://github.com/actions/bin/tree/master/filter

Reply:
Good point 😶 At first I needed to filter by path and had to write my own (I want to avoid basing actions on unofficial third-party sources, by security), but `actor` and `branch` can be just reused!